pkg/local-resource-manager/resource_services.go

```diff
@@ -0,0 +1,153 @@
+package localResourceManager
+
+import (
+	"context"
+	"log"
+
+	nodecorev1alpha1 "fluidos.eu/node/api/nodecore/v1alpha1"
+	reservationv1alpha1 "fluidos.eu/node/api/reservation/v1alpha1"
+	discoverymanager "fluidos.eu/node/pkg/discovery-manager"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/klog/v2"
+	metricsv1beta1 "k8s.io/metrics/pkg/apis/metrics/v1beta1"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+var (
+	scheme = runtime.NewScheme()
+	uids   []string
+)
+
+func init() {
+	_ = metricsv1beta1.AddToScheme(scheme)
+	_ = corev1.AddToScheme(scheme)
+	_ = nodecorev1alpha1.AddToScheme(scheme)
+	_ = reservationv1alpha1.AddToScheme(scheme)
+}
+
+// GetKClient creates a kubernetes API client and returns it.
+func GetKClient(ctx context.Context) (client.Client, error) {
+	config := ctrl.GetConfigOrDie()
+
+	cl, err := client.New(config, client.Options{
+		Scheme: scheme,
+	})
+	if err != nil {
+		klog.Fatalf("error creating manager: %", err)
+	}
+
+	return cl, nil
+}
```

Comment:
check if this is not yet useful

Reply:
Since for the localResourceManager we do not have a manager, yes, it useful...